src/components/Content/Content.tsx

```diff
@@ -79,6 +80,15 @@ const Content = (props: ContentProps): ReactElement => {
 		}
 
 		const fieldName = currentField?.column ? currentField?.column : currentField?.name;
+		let fieldValue = data[fieldName];
```

Comment:
Feel free to resolve this comment because you haven't changed this part but you could eliminate those optional chainings here because we've already done the existence check above.

We could also move the `if (!field) {` check to the top of the method to avoid the `.find()` call & bail out as early as possible.

![image](https://github.com/simpleviewinc/sv-mosaic/assets/13272784/b4e703ce-cd4e-4171-a839-c3e6464921cf)

Reply:
Good call. I'm looking to refactor this component for MOS-1098, so I'll address this change when working on that ticket.